### PR TITLE
General:SPCTR-2372-No need of Negative Line height/Font-Size/Border Radius/Border Width

### PR DESCRIPTION
**Pull request by ravinishadcoder**
### Description
No need of Negative Line height/Font-Size/Border Radius/Border Width
[https://brainstormforce.atlassian.net/browse/SPCTR-2372?atlOrigin=eyJpIjoiYjBiYjI3ZWJmYzYxNDhjOWI2N2M4OTAxMTg0OTc5MzIiLCJwIjoiaiJ9](url)
### Screenshots

[https://d.pr/v/MsfgHj](url)
### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/spacing-control/index.js
Comment (RajkiranBagal) [inline src/components/spacing-control/index.js:1]: This is a very unprofessional way of implementing things. Are we going to determine based on Labels the min value of controls? How many labels we will do this? 

@AbhijitNage123 Please guide the developer here.

File: src/components/typography/range-typography.js
Comment (RajkiranBagal) [inline src/components/typography/range-typography.js:1]: This is a very unprofessional way of implementing things. Are we going to determine based on Labels the min value of controls? How many labels we will do this? 

@AbhijitNage123 Please guide the developer here.

File: readme.txt [hunk 1]
```diff
@@ -175,6 +175,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Inline Notice - Content color is not working in the backend.
 * Fix: Inline Notice - Content typography is not working.
 * Fix: Taxonomy block - Translation missing for Select string in PHP file.
+* General: No need of Negative Line height/Font-Size/Border Radius/Border Width.
```
Comment (RajkiranBagal) [inline readme.txt:178]: Follow our standard practices for adding changelog.
```suggestion
* Fix: Spacing Control - Removed the capability to set negative values for width & radius settings.
* Fix: Typography Control - Removed the capability to set negative values for line-height & font size settings.
```

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/components/spacing-control/index.js [hunk 2]
```diff
@@ -421,24 +429,28 @@ const SpacingControl = ( props ) => {
...
 					className="uagb-spacing-control__number"
 					type="number"
+					min={ props.min }
 					onChange={ ( e ) => onChangeTopValue( e, 'mobile' ) }
 					value={ ( undefined !== valueTopMobile.value ) ? valueTopMobile.value : '' }
 				/>
 				<input
 					className="uagb-spacing-control__number"
 					type="number"
+					min={ props.min }
```
Comment (AbhijitNage123) [inline src/components/spacing-control/index.js:439]: in attribute object set `min = 0` just like `helpText` and use only min. and make sure you have tested it with padding/margin also

File: src/components/typography/range-typography.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ export default function RangeTypographyControl( props ) {
 				{ ...props }
 				label={ props.sizeText }
 				value={ props.size.value || '' }
-				min={ -50 }
+				min={ props.min }
```
Comment (AbhijitNage123) [inline src/components/typography/range-typography.js:26]: let's create a const {} = props and add all attributes that we are taking from props into that const. Refer -https://github.com/brainstormforce/ultimate-addons-for-gutenberg/blob/95f92db9b52f325f0022312bf78214a3bfcc6667/src/components/spacing-control/index.js#L37

File: src/components/spacing-control/index.js (hunk 2 above)
Comment (ravinishadcoder) [reply src/components/spacing-control/index.js]: I have done the suggested changes and tested

File: src/components/typography/range-typography.js (hunk 3 above)
Comment (ravinishadcoder) [reply src/components/typography/range-typography.js]: I have done the suggested changes

**Review by RajkiranBagal — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**